### PR TITLE
Chore: Remove `returnToPrevious` feature toggle

### PR DESCRIPTION
**Pull request by eledobleefe**
**What is this feature?**
- Remove this feature toggle in `registry.go` and related
- Remove the feature toggle check logic in the necessary files.

**Why do we need this feature?**
To remove the RTP feature toggle and decrease the number of feature toggles enabled / managed.

**Who is this feature for?**
Everybody

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/services/featuremgmt/toggles_gen.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
       "metadata": {
         "name": "returnToPrevious",
         "resourceVersion": "1716448665531",
-        "creationTimestamp": "2024-05-23T07:17:45Z"
+        "creationTimestamp": "2024-05-23T07:17:45Z",
+        "deletionTimestamp": "2024-05-27T09:59:33Z"
```
Comment (L-M-K-B) [inline pkg/services/featuremgmt/toggles_gen.json:64]: 🥳

**Review by L-M-K-B — APPROVED** [review]
Looks great from my point of view. I think you caught all the traces of the feature toggle.

**Comment by JoaoSilvaGrafana** [comment]
Do we also want to remove it from the [e2e test](https://github.com/grafana/grafana/blob/main/e2e/various-suite/return-to-previous.spec.ts#L7)?
Only other place I could find it.

**Review by JoaoSilvaGrafana — APPROVED** [review]
lgtm!

**Pull request merged**